### PR TITLE
Bug/fix scheduling duration update after complete

### PR DESCRIPTION
**Pull request by alcorj-mizar**

### CODE REVIEW

File: hack/local-up-volcano.sh [hunk 1]
```diff
@@ -44,6 +45,7 @@ function install-volcano {
 }
 
 function uninstall-volcano {
+  kubectl delete job volcano-admission-init -n $VOLCANO_NAMESPACE
```
Comment (Thor-wl) [inline hack/local-up-volcano.sh:48]: Is this action aimed to prevent that reinstall and bring confliction?

Comment (alcorj-mizar) [reply hack/local-up-volcano.sh:48]: yes, always got this problem when reinstall.

**Comment by Thor-wl** [comment]
@alcorj-mizar Please resolve the confliction

**Pull request merged**